docs/asyncapi-docs/src/test/resources/expected_nested_description_header.yml

```diff
@@ -0,0 +1,33 @@
+asyncapi: 2.0.0
+info:
+  title: Query nested descriptions
+  version: '1.0'
+channels:
+  /:
+    description: Endpoint description
+    subscribe:
+      operationId: onRoot
+      message:
+        $ref: '#/components/messages/oneOf:'
+    publish:
+      operationId: sendRoot
+      message:
+        $ref: '#/components/messages/string'
+    bindings:
+      ws:
+        method: GET
+        headers:
+          type: object
+          properties:
+            Test:
+              type: string
+              description: Test token
+components:
+  messages:
+    string:
+      payload:
+        type: string
+      contentType: text/plain
+    'oneOf:':
```

Comment:
hm this looks suspicious. Where does this come from?

Reply:
It probably comes from this piece of code `webSocketBody[String, CodecFormat.TextPlain, Json, CodecFormat.Json]`

Reply:
hmm but the other test cases don't contain that. This looks incorrect - what's the difference between this one and the others?

Reply:
I think that all examples with `CodecFormat.TextPlain` contains such text for example : `expected_two_endpoints.yml` or `expected_string_json.yml`

Reply:
There's no other occurence of `'oneOf:'` in the sources

Reply:
I check this and found out that `oneOf` is related to `Json, CodecFormat.Json`  and these are only 2 tests with such setup.
If i use exact type ex: `Fruit`  instead of `Json` the `oneOf` is replaced with 
```
Fruit:
      payload:
        $ref: '#/components/schemas/Fruit'
      contentType: application/json
```